src/patternfly/components/Alert/examples/index.js

```diff
@@ -4,15 +4,20 @@ import Example from '@siteComponents/Example';
 
 import AlertTypesRaw from '!raw!./alert-types-example.hbs';
 import AlertVariationsRaw from '!raw!./alert-variations-example.hbs';
+import AlertInlineRaw from '!raw!./alert-inline-example.hbs';
```

Comment:
can you move this above the `AlertVariationsRaw` line that comes before it so `AlertInline` comes before `AlertInlineVariations` throughout this file?